src/en/installconfig-upgrade-1.9-to-2.0.md

```diff
@@ -0,0 +1,47 @@
+Title: Upgrade from 1.9 to 2.0
+
+# Upgrade from 1.9 to 2.0
+
+MAAS 2 is only supported on Ubuntu 16.04 LTS (Xenial Xerus). This is because 
+Ubuntu 16.04 defaults to Python 3, whereas previous versions of Ubuntu defaulted
+to Python 2.
+
+MAAS versions 1.9 and 1.9.x will continue to be supported on Ubuntu 14.04 LTS
+(Trusty Tahr) until they reach [end-of-life](https://wiki.ubuntu.com/Releases).
+
+However, it is possible to upgrade from MAAS 1.9.x to MAAS 2.x as part of the
+distribution upgrade from Ubuntu 14.04 (or Ubuntu 15.10) to Ubuntu 16.04. The
+MAAS database and MAAS configuration will be migrated to the latest MAAS
+version as part of the upgrade process.
+
+Before you start, make sure all your packages are updated (`sudo apt update &&
+sudo apt full-upgrade`). Backups should [also be
+made](https://help.ubuntu.com/14.04/serverguide/backups.html) , and we'd
```

Comment:
Ah. I was thinking of a specific MAAS backup. I thought we had a backup.md page. I may have been thinking of Juju.